Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:16.1-alpine
```

Comment:
Could you swap this to 14.16.1/LTS? Not that I don't want to support 16.x, just waiting for that to flip to LTS before I officially support it.